src/components/carousel/DotsPaginationButton.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FC } from 'react';
+import { chakra, useMultiStyleConfig } from '@chakra-ui/react';
+
+interface DotsButtonProps {
+  isCurrent: boolean;
+  currentDot: number;
+  totalDots: number;
+}
+
+const DotsPaginationButton: FC<DotsButtonProps> = ({
```

Comment:
This is not really a button anymore. Can we go with something like `DotsPaginationIndicator` or `DotsSlideIndicator` ? 🤷